tests/Unit/Util/HmacSignatureTest.php

```diff
@@ -164,7 +164,7 @@ public function testBankingWebhookHmacValidation()
             . "{\"balanceAccountId\":\"BA3227C223222H5HVKT3H9WLC\"},\"currency\":\"EUR\"}},\"environment\":"
             . "\"test\",\"type\":\"balancePlatform.balanceAccountSweep.updated\"}";
         $hmac = new HmacSignature();
-        $result = $hmac->validateHMAC(
+        $result = $hmac->validateHMACSignature(
```

Comment:
Is it possible to keep both unit tests while the method is being deprecated? And then we remove the unit test once fully gone

Reply:
Yes ofcourse I'll commit with both tests